content/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.rst

```diff
@@ -1,84 +1,216 @@
+   :align: center
+   :alt: Enabled barcode setting with Default Nomenclature selected.
 
-.. note::
-        You can define different rules and order their priority thanks to the
-        sequence. The first rule which matches the scanned barcode will be
-        applied.
-
-Configure your Product
-----------------------
-
-1. The barcode of the product should start by “21”;
-2. The 5 “dots” are the other numbers of your product barcode, allowing to identify the product;
-3. The barcode should contain 0’s where you did defined D’s or N’s. In our case we need to set 5 zeros because we configured “21…..{NNDDD}”;
-4. In EAN-13, the last number is a check number, use an EAN13 generator to know which digit it should be in your case.
-
-.. image:: barcode_nomenclature/barcode_nomenclature_03.png
-    :align: center
-
-In case you weight 1,5 Kg of pasta, the balance will print you the
-following barcode 2112345015002. If you scan this barcode in your POS or
-when receiving products in your barcode application, Odoo will
-automatically create a new line for the Pasta product for a quantity of
-1.5 Kg. For the point of sale, a price depending on the quantity will
-also be computed.
-
-.. image:: barcode_nomenclature/barcode_nomenclature_04.png
-    :align: center
-
-Rule Types
-----------
-
-- **Priced Product**: allows you to identify the product and specify its price, used in POS.
-- **Discounted Product**: allows you to create one barcode per applied discount. You can then scan your product in the POS and then scan the discount barcode, discount will be applied on the normal price of the product.
-- **Weighted Product**: allows you to identify the product and specify its weight, used in both POS (in which the price is computed based on the weight) and in inventory.
-- **Client**: allows you to identify the customer, for example used with loyalty program.
-- **Cashier**: allows you to identify the cashier when entering the POS.
-- **Location**: allows you to identify the location on a transfer when multi-location is activated.
-- **Package**: allows you to identify packages on a transfer when packages are activated.
-- **Credit Card**: doesn’t need manual modification, exists for data from the Mercury module.
-- **Unit Product**: allows you to identify a product for both POS and transfers.
+Once the nomenclature is saved and selected, the :menuselection:`Barcode Nomenclatures` settings can
+be accessed, via a hidden menu that is **only** discoverable after enabling :ref:`developer mode
+<developer-mode>`.
+
+Once developer mode is enabled, navigate to :menuselection:`Inventory app --> Configuration -->
+Barcode Nomenclatures` menu, and select :guilabel:`Default Nomenclature`.
+
+From this page, the :guilabel:`Barcode Nomenclature` is specified at the top as the `Default
+Nomenclature`.
+
+Under that, the :guilabel:`|UPC|/|EAN| Conversion` field is set to :guilabel:`Always`, by default.
```

Comment:
The formatting broke here. I suggest to remove the vertical bar symbols `|` from the gui

Reply:
Removed vertical bar symbols - Suggestion implemented ✅